### PR TITLE
Add provider equinix and module_name in Equinix Metal TF configs

### PR DESCRIPTION
**Pull request by vasubabu**

### CODE REVIEW

File: examples/network-edge-device-redundant-connection/versions.tf [hunk 1]
```diff
@@ -4,11 +4,14 @@ terraform {
...
+      version = "~> 1.14"
     }
     azurerm = {
       source  = "hashicorp/azurerm"
       version = ">= 2.72.0"
     }
   }
+  provider_meta "equinix" {
+    module_name = "equinix-fabric-connection-azure/network-edge-device-redundant-connection"
+  }
```
Comment (ocobleseqx) [inline examples/network-edge-device-redundant-connection/versions.tf:16]: Examples don't need to define provider_meta, it's just for the module itself
```suggestion
```

File: examples/service-token-metal-to-azure-connection/versions.tf [hunk 2]
```diff
@@ -4,11 +4,14 @@ terraform {
...
+      version = "~> 1.14"
     }
     azurerm = {
       source  = "hashicorp/azurerm"
       version = ">= 2.72.0"
     }
   }
+  provider_meta "equinix" {
+    module_name = "equinix-fabric-connection-azure/service-token-metal-to-azure-connection"
+  }
```
Comment (ocobleseqx) [inline examples/service-token-metal-to-azure-connection/versions.tf:16]: Examples don't need to define provider_meta, it's just for the module itself
```suggestion
```

File: examples/service-token-redundant-connection/versions.tf [hunk 3]
```diff
@@ -4,11 +4,14 @@ terraform {
...
+      version = "~> 1.14"
     }
     azurerm = {
       source  = "hashicorp/azurerm"
       version = ">= 2.72.0"
     }
   }
+  provider_meta "equinix" {
+    module_name = "equinix-fabric-connection-azure/service-token-redundant-connection"
+  }
```
Comment (ocobleseqx) [inline examples/service-token-redundant-connection/versions.tf:16]: Examples don't need to define provider_meta, it's just for the module itself
```suggestion
```

File: examples/fabric-port-redundant-connection/versions.tf [hunk 4]
```diff
@@ -4,11 +4,14 @@ terraform {
...
+      version = "~> 1.14"
     }
     azurerm = {
       source  = "hashicorp/azurerm"
       version = ">= 2.72.0"
     }
   }
+  provider_meta "equinix" {
+    module_name = "equinix-fabric-connection-azure/fabric-port-redundant-connection"
+  }
```
Comment (ocobleseqx) [inline examples/fabric-port-redundant-connection/versions.tf:16]: Examples don't need to define provider_meta, it's just for the module itself
```suggestion
```

**Review by ocobleseqx — CHANGES_REQUESTED** [review]

**Review by ocobleseqx — CHANGES_REQUESTED** [review]
Please, review requested changes

**Comment by vasubabu** [comment]
> Please, review requested changes

Updated

**Review by ocobleseqx — APPROVED** [review]
👍🏼

**Pull request merged**